src/components/victory-axis/helper-methods.js

```diff
@@ -114,5 +114,17 @@ module.exports = {
       right: [props.width - offset.x, 0]
     }[orientation];
     return `translate(${translate[0]}, ${translate[1]})`;
+  },
+
+  getTickPosition(style, orientation, isVertical) {
+    const orientationSign = { top: -1, left: -1, right: 1, bottom: 1 };
+    const tickSpacing = style.size + style.padding;
```

Comment:
This will result in `NaN` if either `size` or `padding` is undefined. Are we enforcing the existence of those style properties?